src/main/kotlin/com/terraformation/backend/tracking/model/PlantingZoneModel.kt

```diff
@@ -298,14 +298,16 @@ data class PlantingZoneModel<PZID : PlantingZoneId?, PSZID : PlantingSubzoneId?>
   }
 
   /**
-   * Returns the monitoring plot that contains the center point of a shape, or null if the shape
-   * isn't in any plot.
+   * Returns the monitoring plot that is of the correct size and contains the center point of a
+   * shape, or null if the shape isn't in any plot.
    */
   fun findMonitoringPlot(geometry: Geometry): MonitoringPlotModel? {
     val centroid = geometry.centroid
 
     return plantingSubzones.firstNotNullOfOrNull { subzone ->
-      subzone.monitoringPlots.firstOrNull { plot -> plot.boundary.contains(centroid) }
+      subzone.monitoringPlots.firstOrNull { plot ->
+        plot.boundary.contains(centroid) && plot.sizeMeters == MONITORING_PLOT_SIZE_INT
+      }
```

Comment:
This is a pretty reasonable automated comment, but in this case it's wrong because the goal of changing this line is for this function to only return monitoring plots that are usable in new observations, which means we never want 25m ones.